script/husky-commit-lint.js

```diff
@@ -0,0 +1,24 @@
+const chalk = require('chalk')
+const spawn = require('child_process').spawn
+
+const args = process.argv.splice(process.execArgv.length + 2);
```

Comment:
Some semicolon at the end?

Reply:
I was actually surprised there were no semicolons in the first place. It seems we use the `semi: [0, "never"]` rule (so no semicolons expected but errors are ignored), but they're used almost everywhere in the code.

I would personally go for the "always" options for the reasons raised in the airbnb eslint convention: https://github.com/airbnb/javascript#semicolons

Reply:
> semi: [0, "never"]

Hm, never saw any linting errors or notices / warnings about this when we have semicolons. Is our linting setup just throwing warnings?

Reply:
If I remember `0` means that errors are ignored. `1` is a warning and `2` is an error.